pyproject.toml

```diff
@@ -97,6 +97,7 @@ reportlab = "*"
 [tool.coverage.run]
 concurrency = ["gevent"]
 
+
```

Comment:
I don't know why for everyone else these lines get added but for me the commit check seems to delete it.

Reply:
Right. Let me leave it like this for now, but we'll fix this, my (totally unbaked) theory is something happened to this when we integrated the resolver with its pre-commit.